src/components/Navigation/Navigation.jsx

```diff
@@ -37,22 +37,19 @@ export default function Navigation(){
 		}
 	];
 
-	const links      = linkData.map(RENDER.item);
-	const expanded   = open.toString();
+	const links = linkData.map(RENDER.item);
 
 	return (
 		<nav className={ s.wrapper }>
 			<NavToggle />
 			<ul 
 				id="navigation__links"
 				className={ s.container }
-				aria-expanded={ expanded }
+				aria-expanded={ open }
 			>
 				{ links }
 			</ul>
-			<div className={ s.socials }>
-				<Socials />
-			</div>
+			<Socials navOpen={ open } />
```

Comment:
For this, I was thinking something more like `<Socials className={open ? s.open : s.closed} />`
If we wanted the socials to have access to `open` then we'd probably want to do `useContext` inside that component rather than pass it down as a props.

Reply:
Socials now using context as that seemed the nicest way to do it